### DISCUSSION TITLE
oscillating_masses_discrete example infeasible

### DISCUSSION THREAD

**Discussion by szaszrobert** [do-mpc/do-mpc#263] [Q&A]
Dear do-mpc team,

I am trying to get work do-mpc on our own project, however I have bumped into some unexpected difficulties. I implemented an MPC controller to our project, however, the path was not converging towards the origin, so I thought I would go through some do-mpc example setups just to get the hang of it. I have copied the oscillating_masses_discrete example and ran it, however, from the beginning, the problem is marked as infeasible. I even tried to narrow down the starting state x0 range, so that it is closer to the origin, but nothing worked. Maybe I have missed something, or is this a known issue?

Thank you in advance,
Best,
Robert
![oscillating_masses_screenshot](https://user-images.githubusercontent.com/59843285/167103403-1917eb76-6ab2-4a84-aa93-07bfa62c4005.png)

**Comment by 4flixt** [comment]
Hi @szaszrobert ,

It is difficult for me where your problem lies. I just ran both, the Jupyter notebook and the file in ``examples/oscillating_masse_discrete/main.py`` and they worked for me. We also test all example files with travis on each release and have not seen any problems. 

Here are the results that I just received:
<img width="752" alt="Screenshot 2022-05-18 at 16 15 32" src="https://user-images.githubusercontent.com/32864011/169063576-a99777cb-ca16-4d98-8764-662e1a30c877.png">

Best,
4flixt
